docs/auth0_api.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+---
+## auth0 api
+
+Makes an authenticated HTTP request to the Auth0 Management API
+
+### Synopsis
+
+Makes an authenticated HTTP request to the Auth0 Management API and prints the response as JSON.
+
+The method argument is optional, and when you don’t specify it, the command defaults to GET for requests without data and POST for requests with data.
+
+Auth0 Management API Docs:
+  https://auth0.com/docs/api/management/v2
+
+Available Methods:
+  GET, POST, PUT, PATCH, DELETE
+
+```
+auth0 api <method> <uri> [flags]
+```
+
+### Examples
+
+```
+auth0 api "/stats/daily?from=20221101&to=20221118"
```

Comment:
Have we considered using a flag for query params (e.g. `-q "foo=bar" -q "bar=baz"`), instead of passing the entire path + query params already constructed? Asking because having to pass the constructed path + query params means the query params need to be already properly encoded. Whereas if passing them by flag they don't have to be, as the CLI would be tasked with constructing the URL and properly encoding all the bits.

Reply:
The only ever need for query params is to allow for the responses to be paginated or to make sure we include only certain fields in the response. Because of this limited use case the query params were introduced as part of the uri argument. However we could consider switching these to be passed through flags. Wdyt @willvedd as well?

Reply:
@Widcket I played around a bit and ended up allowing both ways of adding query params to coexist: [`329f26c` (#531)](https://github.com/auth0/auth0-cli/pull/531/commits/329f26c54d77c33b50a27cc87db7dfd493fba9a1). Wdyt? 

If we do however want to remove the query params from the uri argument and only allow for them to be passed through the flag we need to change [`329f26c` (#531)](https://github.com/auth0/auth0-cli/pull/531/commits/329f26c54d77c33b50a27cc87db7dfd493fba9a1#diff-9641610d0216306cf67e4b06eecff4cc3ec029d1bdbe4b53a6371b20ccdd9846R188) to `params := url.Values{}`